### PR TITLE
implemented the script

### PR DESCRIPTION
**Pull request by Bemhreth**

### CODE REVIEW

File: user_data/strategies/util.py [hunk 1]
```diff
@@ -1,5 +1,7 @@
 import subprocess
 
+CURRENT_YEAR = 2020
```
Comment (tef-github) [inline user_data/strategies/util.py:3]: rename to BACKTEST_YEAR

File: user_data/strategies/Strategy002.py [hunk 2]
```diff
@@ -167,8 +168,13 @@ def confirm_trade_entry(self, pair: str, order_type: str, amount: float, rate: f
         coin = pair.split("/")[0]
         brain = "Freq_" + self.__class__.__name__
         if IS_BACKTEST:
-            threading.Thread(target=back_tester, args=(current_time, coin, brain)).start()
+            print("Strategy002:current_time = "+str(current_time))
+            if current_time.year != CURRENT_YEAR or current_time.month != CURRENT_MONTH:
```
Comment (tef-github) [inline user_data/strategies/Strategy002.py:172]: Why do we need this logic?
It seems like a hack

If we fix the real cause through the script, we might not need this workaround

**Review by tef-github — CHANGES_REQUESTED** [review]
changes requested

File: user_data/strategies/Strategy002.py (hunk 2 above)
Comment (Bemhreth) [reply user_data/strategies/Strategy002.py:172]: Ow ok so you mean fixing the downloaded json file?

Comment (tef-github) [reply user_data/strategies/Strategy002.py:172]: Yes, that's what I meant yesterday. 

This is also a valid solution, but the logic is not clean:

Use the following:

if currenttime.year is BACKTEST_YEAR and currenttime.month is BACKTEST_MONTH_INDEX + 1:
 threading.Thread
else 
 print ("redundant data")
 pass

Comment (tef-github) [reply user_data/strategies/Strategy002.py:172]: Yes here is the message from yesterday:
"Ok, you can write a script to cut the redundant data out of the file"

Please write a script to cut out redundant data out of the downloaded file.

What you did here is a hack not a fix

Comment (tef-github) [reply user_data/strategies/Strategy002.py:172]: python data_cleaner.py [json_file_name] [month] [year]

python data_cleaner.py abc.json OCT 2021

add this script in root directory of freqtrade repo

Comment (tef-github) [reply user_data/strategies/Strategy002.py:172]: the script will "edit" abc.json

you can define "edit" 
1 "create a new temp file"
2 "delete original"
3 "rename temp to original name"

Comment (tef-github) [reply user_data/strategies/Strategy002.py:172]: please name the script: freq_data_cleaner.py

Comment (Bemhreth) [reply user_data/strategies/Strategy002.py:172]: Ok I understand thanks

**Pull request closed without merging**